scripts/brotli.js

```diff
@@ -1,19 +1,21 @@
 /*
  * Copyright 2020 Adobe. All rights reserved.
- * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * This file is licensed to you under the Apache License, Version 2.0 (the 'License');
  * you may not use this file except in compliance with the License. You may obtain a copy
  * of the License at http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software distributed under
- * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
 
 const fs = require('fs');
 const glob = require('fast-glob');
-const PQueue = require('p-queue').default;
 const zlib = require('zlib');
+// const PQueue = require('p-queue').default;
```

Comment:
no reason to keep this line around with your change right?

Reply:
None at all. I will remove and resubmit.

Cheers!